src/stats/stats.jl

```diff
@@ -58,6 +61,12 @@ function show(io :: IO, stats :: ExecutionStats)
   println("  feasibility measure: $(stats.feas_norm)")
   println("  iterations: $(stats.iter)")
   println("  elapsed time: $(stats.elapsed_time)")
+  if length(stats.solver_specific) > 0
+    println("  solver specifics:")
+    for (k,v) in stats.solver_specific
+      println("    $k: $v")
```

Comment:
The docs say you can also use `@printf` with `io`, but it seems `show` is used to display Julia objects. It's not clear if there's a version of `print` with `io`. I'm thinking that if a solver returns arrays (think of a direction of infinite descent), or Dicts, or whatever, we should display those things as their designer intended.